sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1881,8 +1919,10 @@ if used in application code.
 
 . Using reductions in a graph node.
 . Using sycl streams in a graph node.
-. Profiling an event returned from graph submission with
-  `event::get_profiling_info()`.
+. Profiling information is not available for graphs that contain host-task nodes.
+. Profiling a node from an event returned from graph submission with
+  `event::ext_oneapi_get_profiling_info(ext::node)` is only available for
+  the level-zero backend.
```

Comment:
Again, I think this is surprising.  Removing the per-node profiling completely would fix this.